internal/versions/versions.go

```diff
@@ -101,16 +101,12 @@ func (h *handler) V2ListSupportedOpenshiftVersions(ctx context.Context, params o
 	availableOcpVersions := h.GetOpenshiftVersions()
 	for _, key := range availableOcpVersions {
 		// Fetch available architectures for openshift version
-		availableArchitectures, err := h.GetCPUArchitectures(key)
-		if err != nil {
-			return operations.NewListSupportedOpenshiftVersionsInternalServerError().
-				WithPayload(common.GenerateInternalFromError(err))
-		}
+		availableArchitectures := h.GetCPUArchitectures(key)
 		// Fetch release image for openshift version (values should be similar across architectures)
 		releaseImage, err := h.GetReleaseImage(key, common.DefaultCPUArchitecture)
 		if err != nil {
-			return operations.NewListSupportedOpenshiftVersionsInternalServerError().
-				WithPayload(common.GenerateInternalFromError(err))
+			// Missing release image - skip version
+			continue
```

Comment:
If I understand the case here, maybe we'll be better to release a warning log here?
If the list of release images is missing this image

Reply:
@danielerez can you elaborate on the case here?

Reply:
So this check handles a scenario in which an OS image x.y.z version is specified, but a correlated release image is missing. Refactored this func to go over the release images instead, so it no longer relevant.